waku/v2/node/waku_node.nim

```diff
@@ -258,10 +265,34 @@ proc new*(T: type WakuNode,
     peerManager: PeerManager.new(switch, peerStorage),
     switch: switch,
     rng: rng,
-    enr: enr,
+    enr: populatedEnr,
     announcedAddresses: announcedAddresses
   )
 
+  if discv5Enabled:
+    var discv5Enrs: seq[enr.Record]
+    # parse enrURIs from the configuration and add the resulting ENRs to the discv5BootstrapEnrs seq
+    for enrUri in discv5BootstrapNodes:
+      addBootstrapNode(enrUri, discv5Enrs)
+    discv5Enrs.add(discv5BootstrapEnrs)
+
+    let rawMultiAddrField = enrMultiaddrs.getRawField()
+    # construct openArray[(string, seq[byte])]
+    let discv5EnrMultiAddrs = @[(MULTIADDR_ENR_FIELD, rawMultiAddrField)]
+    wakuNode.wakuDiscv5 = WakuDiscoveryV5.new(
+      extIp, extPort, discv5UdpPort,
+      bindIp,
+      discv5UdpPort.get(),
+      discv5Enrs,
+      discv5EnrAutoUpdate,
+      keys.PrivateKey(nodekey.skkey),
+      wakuFlags.get(),
+      discv5EnrMultiAddrs,
+      wakuNode.rng,
+      DiscoveryConfig.init(discv5TableIpLimit.get(),
```

Comment:
related to my other comments can we pass directly the DiscoveryConfig? should simplify things a bit?

Reply:
Addressed in [185a10a](https://github.com/waku-org/nwaku/pull/1512/commits/185a10ad6637c8c13f2e58c5d7305d290c6a3982), thanks!